sbnobj/Common/Reco/MVAPID.cc

```diff
@@ -0,0 +1,25 @@
+#include "sbnobj/Common/Reco/MVAPID.h"
```

Comment:
I was going to comment that implementation files should be .cxx and that .cc is for modules. But I see we have both conventions within sbnobj already :sob: